### ISSUE TITLE
Add Sightseeing View

### ISSUE THREAD

**Issue by MisterY** [organicmaps/organicmaps#2357]
A follow-up for #1577, specific to the use case of visiting a location as a tourist or a business trip. Please suggest a better name!

The focus is on:

- transportation (stops, lines)
- accommodation
- customizable POIs (as for every view)
- restaurants and bars
- tourist attractions
  - museums
  - ruins, fortifications, castles
  - monuments and art exhibitions in the open
  - galleries
  - etc.

**Comment by MisterY** [comment]
> If there are subtle differences only then it won't justify adding another entity I think.

Yes, I definitely agree on that. As mentioned, it will depend on how useful the POI selection and grouping turns out.
Also, it's a brainstorming suggestion so let's see what turns out of it. Perhaps I should've created it in the Discussions section (can't get used to it, still) because that was the real intention. I don't have a concrete proposal at this stage.

**Comment by j13m126** [comment]
I like the idea of a tourist and local (regular) view. 
the tourist view could have an icon with a human and a trolley in it, the local view an icon with a human in front of a house.

Here is another idea: how about only hiding the icons of irrelevant POIs in tourist/local view, but not the name. The interface would be less cluttered with irrelevant icons while not taking anything away. Take a supermarket: It might not be the most important POI for a tourist, but might still be of value when you want to know where you are or which direction to go.

**Comment by pastk** [comment]
Yes, definitely, we have different means of POIs prioritization: using different icons sizes, leaving only text / only icon, changing zoom levels when POI becomes visible, etc.

**Comment by biodranik** [comment]
@pastk what about removing completely the generic "fit all" map style and focusing on specialized ones? It could greatly simplify everything.

**Comment by pastk** [comment]
Why not? But what will be the default style and how it will be different from the current OM style?
